### ISSUE TITLE
no exported member 'NgModuleMetadataType'

### ISSUE THREAD

**Issue by beakid** [angular-ui/ui-router#3026]
Just installed ui-router in my Angular 2.0 project and immidiately gets this in the console:

```
/Users/admin/Sites/testapp/node_modules/ui-router-ng2/ng2/uiRouterNgModule.d.ts:3:9 
Module '"/Users/admin/Sites/testapp/node_modules/@angular/core/index"' has no exported member 'NgModuleMetadataType'. 
```

Seems NgModuleMetadataType is no longer a thing in Angular 2.0?

**Comment by adharris** [comment]
The various metadata types were removed in RC7.  This was fixed here: 7c54b751bf25f3f07250bc9769917d58b1d30934 but hasn't made it into a release yet.  For now, you'll have to build from master to get RC7 and 2.0.0 final to work.

**Comment by XavierEr** [comment]
Tried to clone the repo and build it locally but I was thrown with this exception.

`Copying typescript sources to C:\Users\Documents\GitHub\ui-router\build_packages\ng2\_src
Excludes: ng1,ng1.ts
--> Staging package...
--> Building webpack bundles...
module.js:457
    throw err;
    ^`
